.gitignore

```diff
@@ -18,3 +18,4 @@ _archive/
 _public/
 
 # Additional entries
+dependencies/cilium/olm/
```

Comment:
This also needs to be added in `.sync.yml` for it to survive modulesync updates.